modules/b2b_logic/logic.c

```diff
@@ -2831,6 +2892,7 @@ int process_bridge_action(struct sip_msg* msg, b2bl_tuple_t* tuple,
 				LM_ERR("Failed to create new b2b entity\n");
 				goto error;
 			}
+			entity->force_send_socket = msg?(msg->force_send_socket?msg->force_send_socket:0):0;
```

Comment:
This line should be removed as there is no such field in the `b2bl_entity_id_t` structure currently.

Reply:
Done. thanks for note. It's a part of another patch.